### DISCUSSION TITLE
Unifi LXC fails during install

### DISCUSSION THREAD

**Discussion by aesir2** [tteck/Proxmox#2243] [Help Needed] [answered]
Hi.
I've used several of your scripts  to lxc up  and running. I'm trying to install the  unifi lxc and picked debian 12 for the install  but once it gets to the point of dns lookup it says "DNS Lookup  Failure" but keeps  going. It can't update the container OS or install any dependencies. I've tried default install and advanced giving it the dns for my pihole (didn't need to for any other scripts) and it fails in either case. I saw in another post that the script is supposed to ask for a user and password but it doesn't ask for either and the only password is for root ssh. I'm running pve 8 and have tried to run it again after updating pve 8 and it is still failing.

Not sure what to do or what the case may be at this point.

edit: new error i got after updating pve8 is "[ERROR] in line 17: exit code 100: while executing command $STD apt-get install -y curl
++ [[ 17 -eq 24 ]]
+ exit"

verbose  mode is on and that output  is ": Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
++ error_handler 17 '$STD apt-get install -y curl'
++ local exit_code=100
++ local line_number=17
++ local 'command=$STD apt-get install -y curl'
++ local 'error_message=\033[01;31m[ERROR]\033[m in line \033[01;31m17\033[m: exit code \033[01;31m100\033[m: while executing command \033[33m$STD apt-get install -y curl\033[m'
++ echo -e '\n\033[01;31m[ERROR]\033[m in line \033[01;31m17\033[m: exit code \033[01;31m100\033[m: while executing command \033[33m$STD apt-get install -y curl\033[m' "

There is also mention of running  apt-get update or --fix-missing but  there's no  way  to do that after the script fails?

**Comment by tteck** [comment]
It seems that you're facing a network problem, _probably_ linked to DNS.

**Reply by aesir2** [reply]
That was my thought too. However when I installed the jellyfin script, it worked without  a hitch. I tried the Debian lxc and ubuntu  lxc and they worked as well.

**Comment by tteck** [comment]
The apps utilizes the same set of reusable scripts, so if one fails, they all should.
https://github.com/tteck/Proxmox/blob/main/CODE-AUDIT.md

**Reply by aesir2** [reply, marked as answer]
Turns out the issue WAS DNS. Apparently there are multiple places in Pfsense that needed it updated to the ip for the adguard lxc to be set by default instead of going into every device and changing it. So not it works just fine. Still puzzling how some worked and some didn't. That's a project for another day.

Thanks for the info. I'll keep that in mind.

**Comment by aesir2** [comment]
It was DNS and pfsense settings. All good now.
